tests/containers/bci_version_check.pm

```diff
@@ -36,6 +36,8 @@ sub run {
         $engine = 'podman';
     } elsif ($engines =~ /docker/) {
         $engine = 'docker';
+    } elsif ($engines =~ /k3s/) {
```

Comment:
Neat picking, but `$engines =~ /(k3s|podman)/` might be a cleaner solution as both conditions have the same outcome, use podman